packages/office-addin-project/src/convert.ts

```diff
@@ -132,4 +134,5 @@ async function updateManifestXmlReferences(): Promise<void> {
   // write updated json to file
   const writeFileAsync = util.promisify(fs.writeFile);
   await writeFileAsync(packageJson, JSON.stringify(content, null, 2));
+  fs.unlinkSync(manifestPath);
```

Comment:
Seems like the wrong place to put this.  The method is about updating the references to the manifest (currently only in the package.json file) . . . deleting it doesn't match.  We are also slightly complicating things by adding a parameter just to make one call to delete the file.

Seems like it really belongs after the convert call in the main convertProject method or in a private method called from the main one that does the convert and file delete.